src/main/java/frc/robot/commands/LiftControl.java

```diff
@@ -42,17 +42,39 @@ public void initialize() {
   /** Called repeatedly when this Command is scheduled to run */
   @Override
   public void execute() {
-    liftControl();
+    switch (Constants.CURRENT_ROBOT) {
+      case MUNCHKIN:
+      liftControlMunchkin();
+      break;
+      case ROBOT2020:
```

Comment:
Monolith also relies on this code, and this would make the lift not work.
The naming of the methods are misleading - doesn't mention that the input methods are different.
Honestly, this code should be moved to a new file, since the buttons and gamepad should be interchangeable.